Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1534,6 +1535,16 @@ bool RenderBox::hitTestClipPath(const HitTestLocation& hitTestLocation, const La
 
     auto offsetFromHitTestRoot = toLayoutSize(accumulatedOffset + location());
     auto hitTestLocationInLocalCoordinates = hitTestLocation.point() - offsetFromHitTestRoot;
+
+    auto hitsClipContent = [&](Element& element) -> bool {
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+        if (auto* clipper = dynamicDowncast<RenderSVGResourceClipper>(element.renderer()))
+            return !clipper->hitTestClipContent(FloatRect(borderBoxRect()), hitTestLocationInLocalCoordinates);
```

Comment:
Does it need implicit FloatRect conversion? Didn't check if that constructor taking a LayoutRect is 'explicit' or not. Either way, you could try { borderBoxRect() } instead in that case.

Reply:
That seems to work.